marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/SortingStrategyConfiguration.kt

```diff
@@ -43,10 +43,11 @@ class ExecutionTimeSortingStrategyConfiguration {
 
 class SuccessRateSortingStrategyConfiguration {
     var limit: Instant = Instant.now().minus(DEFAULT_DAYS_COUNT, ChronoUnit.DAYS)
+    var reverse: Boolean = false
```

Comment:
I would name this `ascending` (or `descending`) for clarity

Reply:
Good call. Done